itests/kit/ensemble.go

```diff
@@ -260,7 +260,7 @@ func (n *Ensemble) MinerEnroll(minerNode *TestMiner, full *TestFullNode, opts ..
 		)
 
 		// Will use 2KiB sectors by default (default value of sectorSize).
-		proofType, err := miner.SealProofTypeFromSectorSize(options.sectorSize, n.genesis.version, false)
+		proofType, err := miner.SealProofTypeFromSectorSize(options.sectorSize, n.genesis.version, miner.SealProofVariant_Standard)
```

Comment:
Why are we always using `Standard` in the ensemble ? What if tests wanna use NI ?

Reply:
then someone's going to have to add it - this is for the full lotus-miner and it has no support for dealing with NI, it assumes precommit for a start, so currently it doesn't make sense to parameterise this